package.json

```diff
@@ -3,88 +3,89 @@
   "version": "3.9.0",
   "license": "Apache-2.0",
   "scripts": {
-    "build": "export NODE_OPTIONS=\"--max-old-space-size=4096 $NODE_OPTIONS\"; stripes build stripes.config.js",
-    "stripes": "stripes",
-    "start": "stripes serve stripes.config.js",
-    "build-module-descriptors": "stripes mod descriptor stripes.config.js --output ./ModuleDescriptors",
-    "local": "f=stripes.config.js; test -f $f.local && f=$f.local; echo Using config $f; stripes serve $f",
-    "test": "echo 'No unit tests implemented'",
-    "lint": "eslint test/ui-testing"
```

Comment:
I think we should keep it like in release branch

Reply:
In R1-2025 we set next:
https://github.com/folio-org/platform-complete/blob/R1-2025/package.json#L6
I took this block from snapshot branch with --max-old-space-size=8000